src/app/components/client/dialog/Dialog.tsx

```diff
@@ -44,9 +50,18 @@ export const Dialog = ({
         {...dismissButtonProps}
         ref={dismissButtonRef}
         className={styles.dismissButton}
-        // TODO: Add unit test when changing this code:
-        /* c8 ignore next */
-        onClick={() => onDismiss()}
+        // Not all dialogs have telemetry exit events
```

Comment:
I think this comment is meant for a couple of lines down, for the `dismissalTelemetryId`, no?